frontend/src/store/modules/shoots/getters.js

```diff
@@ -150,38 +151,51 @@ export default {
     return state.initialNewShootResource
   },
   searchItems (state, getters, rootState, rootGetters) {
+    const searchPattern = /(-?"[^"]+")*([\S]+)*/g
+    const termPattern = /^(-)?("?)([^"]+)("?)$/g
```

Comment:
add tests for the regex so that we can be sure in the future that it still works as expected in case we need to adapt it